src/__tests__/pages/policy/input/ParameterEditor.test.js

```diff
@@ -0,0 +1,61 @@
+// src/components/ParameterEditor.test.jsx
+
+import React from "react";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import ParameterEditor from "../../../../pages/policy/input/ParameterEditor";
+import { BrowserRouter } from "react-router-dom";
+
+global.URL.createObjectURL = jest.fn();
+
+jest.mock("react-plotly.js", () => {
+  return {
+    __esModule: true,
+    default: () => <div data-testid="mock-plot">Mock Plot Component</div>,
+  };
+});
+
+const metadata = {
+  parameters: {
+    exampleParameter: {
+      label: "Example Parameter",
+      description: "This is an example parameter.",
+      period: "monthly",
+      values: {
+        "2024-01-01": 10,
+        "2024-02-01": 20,
+      },
+      unit: "bool",
+    },
+  },
+  countryId: "exampleCountryId",
```

Comment:
Could you actually set this one to either "us" or "uk"? That makes it a bit more similar to the actual functioning.

Reply:
In the new commit I have added the countryId as 'us'.